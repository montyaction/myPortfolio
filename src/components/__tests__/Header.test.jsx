import React from "react";
import { render } from "@testing-library/react";
import Header from "../Header";

test('renders correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
});