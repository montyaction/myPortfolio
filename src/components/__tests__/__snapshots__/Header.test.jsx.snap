// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <header>
    <nav
      class="nav"
      id="navbar"
    >
      <ul
        class="nav-list"
      >
        <li>
          <a
            href="#welcome-section"
          >
            About
          </a>
        </li>
        <li>
          <a
            href="#projects"
          >
            Work
          </a>
        </li>
        <li>
          <a
            href="#contact"
          >
            Contact
          </a>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
